SofaKernel/SofaBase/SofaBaseConfig.cmake.in

```diff
@@ -4,25 +4,26 @@
 @PACKAGE_INIT@
 
 set(SOFABASE_TARGETS @SOFABASE_TARGETS@)
-set(SOFAEIGEN2SOLVER_HAVE_OPENMP @SOFAEIGEN2SOLVER_HAVE_OPENMP@)
 
 find_package(SofaFramework QUIET REQUIRED)
 find_package(SofaSimulation QUIET REQUIRED)
 
-# Eigen3 is required by SofaBaseTopology and SofaEigen2Solver
-find_package(Eigen3 QUIET REQUIRED)
+# Temporary list while doing packaging
+set(SOFABASE_MODULES SofaBaseLinearSolver SofaEigen2Solver SofaBaseTopology SofaBaseCollision SofaBaseUtils)
```

Comment:
Eigen3 is automatically added with SofaEigen2Solver?

Reply:
Yes